script_editor/app/views/plays/show.html.erb

```diff
@@ -125,6 +125,51 @@
             <!-- This button shows the play's analytics -->
             <button class="nav-analytics" id="analytics-button">ANALYTICS</button>
 
+            <!-- This button toggles the strikethrough edits' visibility -->
+            <button class = "nav-synopsis" curState = "showing" id="viewToggle-button"> Hide Edits </button>
+
+            <script>
+                var toggleBtn = document.getElementById("viewToggle-button");
+                toggleBtn.curState = "showing";
+                toggleBtn.onclick = function() {
+                    var words = Array.from(document.getElementsByClassName("word"));
+                    var punctuation = Array.from(document.getElementsByClassName("punc"));
+                    var directions = Array.from(document.getElementsByClassName("stage"));
+                    var cuttableScript = words.concat(punctuation).concat(directions);
+                    cuttableScript.forEach(function(word) {
+                        if (toggleBtn.curState == "showing") // User wishes to hide edit
```

Comment:
Nesting ifs like this is ok in my opionion, but I was wondering if as a team we considered this style of conditional assigning to something like
`word.style.display = (toggleBtn.curState == "showing && word.dataset.cut == "true") ? 'none' : 'initial';`

These should do the same thing, I'm just curious which we consider to be more clear.

Reply:
I think your first idea of if (a && B) is good and more readable than this form, but this form is more savvy so it depends on what we are going for. 
I tend to go for more readable rather than elegant but thats a personal preference